applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py

```diff
@@ -262,6 +270,19 @@ def __init__(self, model, custom_settings):
         ## Construct the linear solver
         self.linear_solver = linear_solver_factory.ConstructSolver(self.settings["linear_solver_settings"])
 
+        if not self.settings["turbulence_model"].IsEquivalentTo(KratosMultiphysics.Parameters("{}")):
+            # if not empty
+            if CheckIfApplicationsAvailable("RANSModellingApplication"):
+                import KratosMultiphysics.RANSModellingApplication as KratosRANS
```

Comment:
Please import everything at the beginning of the file, if possible. (I would suggest conditionally importing at the beginning and here throwing the error if "not available".